garak/generators/huggingface.py

```diff
@@ -87,6 +90,10 @@ def _load_client(self):
             self.generator.tokenizer = AutoTokenizer.from_pretrained(
                 pipeline_kwargs["model"]
             )
+        self.use_chat = (
+            hasattr(self.generator.tokenizer, "chat_template")
+            and self.generator.tokenizer.chat_template is not None
+        )
```

Comment:
Should we expand this to allow the user to override `use_chat`?

This would defer to the `Configurable` loaded value if provided via config:
```suggestion
        if not hasattr(self, "use_chat"):
            self.use_chat = (
                hasattr(self.generator.tokenizer, "chat_template")
                and self.generator.tokenizer.chat_template is not None
            )
```